official/benchmark/models/resnet_imagenet_test.py

```diff
@@ -236,7 +248,10 @@ def test_end_to_end_xla_2_gpu_fp16(self, flags_key):
     extra_flags = extra_flags + self.get_extra_flags_dict(flags_key)
```

Comment:
A larger block of code is repeated 3 times, could it be extracted into its own function?

Reply:
hmm, will need some refactoring perhaps.